.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/dmlc/ps-lite
 [submodule "3rdparty/dlpack"]
 	path = 3rdparty/dlpack
-	url = https://github.com/dmlc/dlpack
+	url = https://github.com/ElaineBao/dlpack.git
```

Comment:
Will need to be changed once the changes are merged to upstream dlpack. Leaving this comment as a reminder ;-).

Reply:
Definitely :) We're working on PR the related code in dlpack https://github.com/dmlc/dlpack/issues/45